extensionsGallery.json

```diff
@@ -4755,6 +4755,71 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "93",
+					"extensionName": "column-aligner-ads",
+					"displayName": "Column Aligner",
+					"shortDescription": "This extension provides a shortcut key to consistently align the selected rows to the nearest tabstop",
+					"publisher": {
+						"displayName": "Richard Tallent",
+						"publisherId": "RichardTallentVS",
+						"publisherName": "Richard Tallent"
+					},
+					"versions": [
+						{
+							"version": "1.0.0",
+							"lastUpdated": "10/23/2022",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://tallent.us/column-aligner-ads/column-aligner-ads-1.0.0.vsix"
```

Comment:
Unfortunately we can't have direct VSIX links to non-MS extensions. You'll need to change the asset type to `Microsoft.SQLOps.DownloadPage`

Reply:
Changed the assetType, but are you saying I need a landing page rather than a link directly to the VSIX file as well?

Reply:
Ah! All done now.